I see it fails without this change:

```
...
gcc -ansi -pedantic -DLITTLE_ENDIAN -O6 -IParser -IStandalone -IMisc -ILibOneChannel -ILibMultiChannel -IEngine -IDatabase -Wall -DROMDATABASE_STORE -DROMDATABASE_INIT -DSIGNAL  -o Bin/Standalone/Standalone/synth.o -c Standalone/synth.c
/nix/store/v1d8l3zqnia3hccqd0701szhlx22g54z-bash-5.1-p8/bin/bash: line 1: gcc: command not found
make: *** [Makefile:150: Bin/Standalone/Standalone/synth.o] Error 127
builder for '/nix/store/5vlkm9r3hdyzgfkivymb3vxq5dbxa0i2-mbrola-riscv64-unknown-linux-gnu-3.3.drv' failed with exit code 2
error: build of '/nix/store/5vlkm9r3hdyzgfkivymb3vxq5dbxa0i2-mbrola-riscv64-unknown-linux-gnu-3.3.drv' failed
```

and works with this change:

```
[davidak@gaming:~/code/nixpkgs]$ nix-build . -A pkgsCross.riscv64.mbrola
...
post-installation fixup
shrinking RPATHs of ELF executables and libraries in /nix/store/bv00bp3k71q8p8gp712n83d914kq31iq-mbrola-riscv64-unknown-linux-gnu-3.3
shrinking /nix/store/bv00bp3k71q8p8gp712n83d914kq31iq-mbrola-riscv64-unknown-linux-gnu-3.3/bin/mbrola
riscv64-unknown-linux-gnu-strip is /nix/store/n4pyn69crs68zmidxhacmmsqajs982jn-riscv64-unknown-linux-gnu-stage-final-gcc-debug-wrapper-10.3.0/bin/riscv64-unknown-linux-gnu-strip
stripping (with command riscv64-unknown-linux-gnu-strip and flags -S) in /nix/store/bv00bp3k71q8p8gp712n83d914kq31iq-mbrola-riscv64-unknown-linux-gnu-3.3/bin
patching script interpreter paths in /nix/store/bv00bp3k71q8p8gp712n83d914kq31iq-mbrola-riscv64-unknown-linux-gnu-3.3
checking for references to /build/ in /nix/store/bv00bp3k71q8p8gp712n83d914kq31iq-mbrola-riscv64-unknown-linux-gnu-3.3...
/nix/store/bv00bp3k71q8p8gp712n83d914kq31iq-mbrola-riscv64-unknown-linux-gnu-3.3
```

--- a/pkgs/applications/audio/mbrola/default.nix
+++ b/pkgs/applications/audio/mbrola/default.nix
@@ -19,6 +19,9 @@ stdenv.mkDerivation rec {
     sha256 = "1w86gv6zs2cbr0731n49z8v6xxw0g8b0hzyv2iqb9mqcfh38l8zy";
   };
 
+  # required for cross compilation
+  makeFlags = [ "CC=${stdenv.cc.targetPrefix}cc" ];
+
   installPhase = ''
     runHook preInstall
     install -D Bin/mbrola $out/bin/mbrola

